concourse/client.py

```diff
@@ -394,7 +509,7 @@ class ModelBase(object):
     Not intended to be instantiated by users of this module
     '''
 
-    def __init__(self, raw_dict: dict, concourse_api:ConcourseApi):
+    def __init__(self, raw_dict: dict, concourse_api: ConcourseApiBase):
```

Comment:
could you add an `@ensure_annotations`. Was missing before, but since you are touching the coding anyways..

Reply:
done